File: src/www/__test__/testSteps/alert/DismissTheAlert.ts
import { AbstractPostLineStep } from "../AbstractPostLineStep";
import { dismissAlert } from "./helpers";

export class DismissTheAlert extends AbstractPostLineStep {
  constructor() {
    super(/Dismiss the alert/i);
  }

  async runMatch(_: any, match: string[]) {
    dismissAlert();
  }
}

File: src/test/java/com/drpicox/game/testSteps/alert/DismissTheAlert.java
package com.drpicox.game.testSteps.alert;

import com.drpicox.game.testPost.reader.PostLine;
import com.drpicox.game.testSteps.AbstractPostLineStep;
import org.springframework.stereotype.Component;

@Component
public class DismissTheAlert extends AbstractPostLineStep {

    @Override
    protected String getRegex() {
        return "Dismiss the alert";
    }

    @Override
    protected void run(PostLine line, String[] match) {
        // No implementation required by tests in backend
    }
}
